luaui/Widgets/gfx_DrawUnitShape_GL4.lua

```diff
@@ -60,7 +60,7 @@ local unitShaderConfig = {
 local unitShapeShaderConfig = {
 	STATICMODEL = 1.0, -- do not touch!
 	TRANSPARENCY = 0.5,
-	SKINSUPPORT = Spring.Utilities.EngineVersionAtLeast(105,1,1,1653) and 1 or 0,
+	SKINSUPPORT = Script.IsEngineMinVersion(105, 0, 1653) and 1 or 0,
```

Comment:
The middle has to be 0 which I think is an engine bug, so I did this change as a PR instead of just a push. Realistically it seems redundant anyway (is always set to 1 in actual builds) so I think it's not a big deal.